{
  "vid": "a23871f6-059b-11eb-8758-e0d55e2a8bf9",
  "topic": "upnp -- denial of service (crash)",
  "affects": [
    {
      "name": [
        "upnp"
      ],
      "range": [
        {
          "lt": "1.12.1_1,1"
        }
      ]
    }
  ],
  "description": {
    "xmlns": "http://www.w3.org/1999/xhtml",
    "text": "\n\t<p>CVE mitre reports:</p>\n\t<blockquote cite=\"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13848\">\n\t  <p>Portable UPnP SDK (aka libupnp) 1.12.1 and earlier allows remote attackers to cause a denial of service (crash) via a crafted SSDP message due to a NULL pointer dereference in the functions FindServiceControlURLPath and FindServiceEventURLPath in genlib/service_table/service_table.c. </p>\n\t</blockquote>\n      "
  },
  "references": {
    "url": [
      "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-13848",
      "https://nvd.nist.gov/vuln/detail/CVE-2020-13848",
      "https://github.com/pupnp/pupnp/issues/177",
      "https://github.com/pupnp/pupnp/commit/c805c1de1141cb22f74c0d94dd5664bda37398e0"
    ],
    "cvename": [
      "CVE-2020-13848"
    ]
  },
  "dates": {
    "discovery": "2020-06-04",
    "entry": "2020-10-03"
  }
}
